documentation/modules/con-securing-client-labels.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// assembly-securing-kafka-clients.adoc
+
+[id='con-securing-client-labels-{context}']
+= Identifying a Kafka cluster for user handling
+
+A `KafkaUser` resource includes a label that defines the appropriate name of the Kafka cluster (derived from the name of the `Kafka` resource) to which it belongs.
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersion}
+kind: KafkaUser
+metadata:
+  name: my-user
+  labels:
+    strimzi.io/cluster: my-cluster
+----
+
+The label is used by the User Operator to identify the `KafkaUser` resource and create a new user, and also in subsequent handling of the user.
+
+If the label does not match the Kafka cluster, the User Operator cannot identify the `KafkaUser` and the user is not created.
```

Comment:
Maybe we can add that if the `.status` section of the KafkaUser resource is empty after several minutes, users should double check the label?

Reply:
Shall we add a check too:

kubectl get KafkaUser my-user -o jsonpath='{.status}'